docs/agents/sematext-agent/kubernetes/installation.md

```diff
@@ -144,6 +212,17 @@ spec:
           - name: http
             containerPort: 80
             protocol: TCP
+      - name: logagent
+        image: sematext/logagent:latest
+        imagePullPolicy: Always
+        - name: LOGS_TOKEN
+          value: <YOUR_LOGS_TOKEN>
+        - name: REGION
+          value: "US"
```

Comment:
Can we use <"US" or "EU"> so this is self-documenting? People will need to add their tokens anyway, even if they copy-paste this.